prow/config/prow-config-documented.yaml

```diff
@@ -904,6 +904,31 @@ plank:
 pod_namespace: ' '
 
 
+# ProwJobDefaultEntries holds a list of defaults for specific values
+# Each entry in the slice specifies Repo and CLuster regexp filter fields to
+# match against the jobs and a corresponding ProwJobDefault . All entries that
+# match a job are used. Later matching entries override the fields of earlier
```

Comment:
This seems prone to surprising behavior (aka today's job gets sent tomorrow to a new and wrong tenant because someone adds a new item to this list).

Did you consider ensuring that list items to not conflict with each other?

Reply:
We actually want some list items to conflict with each other in some cases. For example myOrg has a default tenantID: foo but myOrg/publicRepo has tenantID: bar.

We will want to add presubmit validation to the config to ensure that the tenantID for each prowjob is what we are expecting for each tenant to increase confidence that we don't accidentally change a private prowjob to a public tenantID